.evergreen/config.yml

```diff
@@ -82,11 +82,60 @@ functions:
         command: |
           astrolabevenv/${PYTHON_BIN_DIR}/astrolabe spec-tests run-one tests/${TEST_NAME}.yaml -e integrations/${DRIVER_DIRNAME}/workload-executor
 
+  "validate executor":
+    # Run a MongoDB instance locally.
+    - command: shell.exec
+      params:
+        working_dir: astrolabe-src
+        script: |
+          set -o xtrace
```

Comment:
This section looks like a copy of `run-mongodb.sh`, is that a mistake?

Reply:
Moved everything to `run-mongodb.sh`.